test/v1/models/response/ListBSASResponseSpec.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response
+
+import play.api.libs.json.{JsSuccess, Json}
+import support.UnitSpec
+import v1.fixtures.ListBSASFixtures._
+import v1.models.domain.TypeOfBusiness
+import v1.models.response.common.{AccountingPeriodResponse, BSASEntries, BusinessSourceSummary}
+
+class ListBSASResponseSpec extends UnitSpec {
+
+  val model =
+    ListBSASResponse(
+      Seq(BusinessSourceSummary(
```

Comment:
This BusinessSourceSummary is the same as the one in BusinessSourceSummarySpec so you could just pull that model into a Fixture and import it

Reply:
It isn't quite the same, because it's not wrapped in the array